index.js

```diff
@@ -1725,4 +1816,27 @@ function getExtension (db, opts) {
   return Extension.register(db)
 }
 
+async function isLinked (batch, block) {
+  const seq = block.seq
+  block.inflate()
+
+  const keys = [block.key]
+
+  for (const l of block.index.levels) {
+    for (const c of l.children) {
+      if (c.seq !== seq) continue
+      const lvl = block.index.levels[c.offset]
```

Comment:
This code is touching storage internals that I'm not familiar with, so I'm skipping the review for this (I don't know what the `levels` and `keys` are here)

Reply:
All good, maf knowledge atm…